prisma/schema.prisma

```diff
@@ -15,25 +15,28 @@ enum Role  {
 
 model User {
     id String @id
+    name String
     createdAt DateTime @default(now())
     updatedAt DateTime @default(now())
     role Role @default(USER)
     ratings Rating[] @relation("Ratings")
+    ratingCount Int @default(0)
+    ratingTotal Int @default(0)
```

Comment:
What is the purpose of this field?

Reply:
It's just a way to quickly calculate the rating average for the user ( ratingTotal / ratingCount = Average Rating)
It's not really necessary if there are only a few ratings, but if there are thousands of ratings it might make the calculation several milliseconds quicker, because you only have to fetch two values from the database instead of calculating everything from scratch.